README.md

```diff
@@ -154,7 +154,7 @@ print(balance) // Logs a balance in drops of XRP
 
 ### Checking Transaction Status
 
-A `XRPClient` can check the status of an transaction on the XRP Ledger. 
+A `XRPClient` can check the status of an transaction on the XRP Ledger.
```

Comment:
This should read "An `XRPClient` can check the status of a transaction on the XRP Ledger." (Getting our "a" v.s. "an" correct so we don't sound dumb)

Reply:
Done